#!/usr/bin/env python

# set up parameters that we care about
PACKAGE = 'jsk_pcl_ros'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("queue_size", int_t, 0, "queue size", 100, 1, 1000);

exit(gen.generate(PACKAGE, PACKAGE, "PrimitiveShapeClassifier"))
